schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -165,45 +165,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-03-01/Microsoft.Network.json#/resourceDefinitions/trafficManagerProfiles" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-05-01/Microsoft.Network.json#/resourceDefinitions/trafficManagerProfiles" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Network.json#/resourceDefinitions/trafficManagerProfiles" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-08-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts" },
```

Comment:
Why remove storage from this file?

Reply:
In autogeneration pipeline, the refs are added in autogeneratedResources.json, and then referenced back to this file.